src/leaderboardPage.js

```diff
@@ -1,49 +1,69 @@
-import React from 'react';
+import React, {Component} from 'react'
 import './App.css';
-import { FaArrowCircleLeft } from 'react-icons/fa';
-import { Link } from 'react-router-dom';
+import {FaArrowCircleLeft} from 'react-icons/fa';
+import {Link} from 'react-router-dom';
 import * as ReactBootstrap from 'react-bootstrap';
+import axios from 'axios';
 
-function leaderboardPage() {
-    const players = [
-      {rank: "1", name: "Nam", score: 31223},
-      {rank: "2", name: "Michael", score: 29943},
-      {rank: "3", name: "Edward", score: 27181},
-      {rank: "4", name: "Brett", score: 19044},
-      {rank: "5", name: "Will", score: 19021},
-    ]
-  
-    const renderPlayer = (player, index) => {
-      return(
+const leadersBackup = [
+    {name: "Nam", score: 31223},
+    {name: "Michael", score: 29943},
+    {name: "Edward", score: 27181},
+    {name: "Brett", score: 19044},
+    {name: "Disconnected", score: 19021},
+]
+
+const renderPlayer = (player, index) => {
+    return (
         <tr key={index}>
-          <td>{player.rank}</td>
-          <td>{player.name}</td>
-          <td>{player.score}</td>
+            <td>{index + 1}</td>
```

Comment:
Good adaptation to the new ranking system. It was smart to get rid of the server-sided rankings as it streamlined the communication with the database.